core/backend/src/GeoCoordServices.ts

```diff
@@ -0,0 +1,21 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import { IModelJsNative } from "@bentley/imodeljs-native";
+import { Range2dProps } from "@itwin/core-geometry";
+
+/** Class containing services for working with geographic coordinates.
+ * @public
+ */
+export abstract class GeoCoordServices {
+  /** Get a list of Geographic Coordinate Systems.
```

Comment:
We use both GCS (geographic coordinate system) and CRS (coordinate reference system) in our docs and APIs...CRS is the more common term, isn't it?

Reply:
There is already a "GeoCoordConfig.ts" file / class in the same directory. I would hesitate to use another term at this level. I think this would bring confusion.

Reply:
Do you mean to change GCS of CRS in every file (and file name) that I touched ? Because it creates a bit of confusion is some cases since "GCS" is already used for some base types:

bvector<CRSListResponseProps> GeoServicesInterop::GetListOfCRS(bool ignoreLegacy, DRange2dCP extent)
    {
    static GeoCoordinates::BaseGCSPtr ll84CRSPtr = GeoCoordinates::BaseGCS::CreateGCS("LL84");
    bvector<CRSListResponseProps> listOfCRS;

    char csKeyName[128];
    CRSListResponseProps props;
    for (int index = 0; (0 < GeoCoordinates::CSMap::CS_csEnum(index, csKeyName, sizeof(csKeyName))); index++)
        {
        GeoCoordinates::BaseGCSPtr crs = GeoCoordinates::BaseGCS::CreateGCS(csKeyName);
...

Reply:
We don't care about filenames nor about the C++ APIs, only the core-backend APIs.

Reply:
I'm now using "CRS" in the core-backend APIs.